tests/fixtures/feature_overlap.json

```diff
@@ -0,0 +1,46 @@
+{
+  "compareFunction": "feature_overlap",
+  "fixtures": [
+    {
+      "description": "feature_overlap return result",
+      "newVersion": {
+        "type": "Feature",
+        "properties": {
+          "osm:version" : 1,
+          "natural": "water"
+        },
+        "geometry": {
+          "type": "Polygon",
+          "coordinates": [
+            [
+              [
+                77.55291938781738,
+                12.984161807639202
+              ],
+              [
+                77.55291938781738,
+                12.9838324899327
+              ],
+              [
+                77.55411028862,
+                12.983774989970945
+              ],
+              [
+                77.55415320396423,
+                12.984083398701047
+              ],
+              [
+                77.55291938781738,
+                12.984161807639202
+              ]
+            ]
+          ]
+        }
+      },
+      "oldVersion": {},
+      "expectedResult": {
+        "result:feature_overlap": 24
+      }
+    }
+  ]
```

Comment:
Yes, that's sounds like a good idea, we should have test for falsey results as well.